client/app/onboarding/pci/pci.html

```diff
@@ -0,0 +1,26 @@
+<div class="cui-page__content">
+    <oui-slideshow data-theme="pci">
+        <oui-slideshow-panel data-heading="{{ ::'pci_slideshow_heading1' | translate }}" data-picture="../../assets/images/onboarding/revamp-PCI.jpg">
+            <ul>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_infra1"></li>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_infra2"></li>
+            </ul>
```

Comment:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/list doesn't fit the needs ?